sonar-project.properties

```diff
@@ -10,4 +10,17 @@ sonar.language=cpp
 sonar.cfamily.build-wrapper-output=bw-outputs
 
 # Encoding of the source files
-sonar.sourceEncoding=UTF-8
\ No newline at end of file
+sonar.sourceEncoding=UTF-8
+
+# Path to the tests
+sonar.tests=tests/data
```

Comment:
Why do you need this?

Reply:
It will allow us to specify the path of our tests for future analysis configuration.. We will see

Reply:
Maybe tests is enough and not tests/data... But we will see in a later PR.